en/controllers/middleware.rst

```diff
@@ -45,11 +70,11 @@ hook method will be called early in the request process, you can use the
 
     class Application extends BaseApplication
     {
-        public function middleware($middlewareStack)
+        public function middleware($middleware)
```

Comment:
I prefer the it without or with `Stack`... Queue is in the "people will miss-type it" ballpark.